tests/e2e/autoscale/02-install.yaml

```diff
@@ -0,0 +1,37 @@
+# This updates an existing deployment with a new Spec.
```

Comment:
Thanks for outlining the steps for me @iblancasa - this helped me understand how the e2e test is working a lot better. The final step waiting for scale down failed in the e2e tests and I realized that I had to add a timeout (more than 5 minutes) because it seems the HPA can take a lot of time to finally scale down.  I've tried modifying the `scaleDown` behavior but sadly I have not been able to get scale down to happen faster.

Reply:
For reference this is what I tried adding to remove the 6 minute timeout, but this doesn't seem to have the intended effect.
```
    behavior:
      scaleDown:
        policies:
        - type: Pods
          value: 1
          periodSeconds: 10
        stabilizationWindowSeconds: 1
```

Reply:
Ah nevermind I now see the issue is that the behavior is not set when the HPA is updated, so I had to add this to the original 0-install.yaml. I created an[ issue](https://github.com/open-telemetry/opentelemetry-operator/issues/1516) for this and have a followup PR in progress. @iblancasa Let me know if you have any feedback on the added script and testing steps - thank you!